### ISSUE TITLE
SEVERE ERROR line 206 in file symini_ftn.n, illegal ilm, invalid atyp and invalid dtype

### ISSUE THREAD

**Issue by pawosm-arm** [flang-compiler/flang#1278]
Since QUADFP was introduced I am observing the following warnings while building flang:

```
SEVERE ERROR line 206 file tools/flang2/utils/symtab/symini_ftn.n: bad atyp, assumed to be DT_INT
SEVERE ERROR line 206 file tools/flang2/utils/symtab/symini_ftn.n: bad dtype, assumed to be DT_INT
WARNING line 206 file tools/flang2/utils/symtab/symini_ftn.n: illegal ilm, QCONJG, in template
```

Although this is only a warning and does not interrupt building process, it would be good to investigate its origin.

**Comment by bryanpkc** [comment]
This is caused by an accidental addition of `QCONJG` to the file, which should have been part of the quad-precision complex type support (to be upstreamed in a future PR). `QCONJG` is not used anywhere currently; I'll submit a PR to revert this addition.

**Comment by bryanpkc** [comment]
Actually, there are other uses of the enum values `I_QCONJG` and `IM_QCONJG`. I think there is a better, simpler way to fix the warning.

**Comment by bryanpkc** [comment]
I have added a fix for this problem to #1215:

https://github.com/flang-compiler/flang/pull/1215/commits/bd978a7bfd408ff5bc8434731a6df54bcf586ac2

**Comment by bryanpkc** [comment]
The fix has been merged into flang/master.
